### ISSUE TITLE
Graphical bug

### ISSUE THREAD

**Issue by lucasimons** [MindFreeze/ha-sankey-chart#74]
Hi i have found this bug if the passthrough are on top of list

**Checklist:**

- [ x] I updated to the latest version available
- [ x] I cleared the cache of my browser

my card code:
```
type: custom:sankey-chart
title: Energia mensile
show_names: true
round: 1
unit_prefix: k
wide: true
min_box_distance: 5
min_box_height: 3
sections:
  - entities:
      - entity_id: sensor.monthly_pvenergy
        name: FV1
        color: rgb(255,215,0)
        children:
          - sensor.monthly_inverter
          - sensor.monthly_dc_energy
          - sensor.monthly_battenergycharged
      - entity_id: sensor.monthly_fvenergy
        name: FV2
        color: rgb(255,105,0)
        children:
          - sensor.monthly_inverter
          - sensor.monthly_dc_energy
          - sensor.monthly_battenergycharged
      - entity_id: sensor.monthly_battenelenergycharged
        name: ENEL IN
        color: purple
        children:
          - sensor.monthly_battenergycharged
  - entities:
      - entity_id: sensor.monthly_inverter
        type: passthrough
        children:
          - sensor.monthly_inverter
      - entity_id: sensor.monthly_dc_energy
        type: passthrough
        children:
          - sensor.monthly_dc_energy
      - entity_id: sensor.monthly_battenergydischarged
        name: OUT
        color: rgb(225,25,0)
        children:
          - sensor.monthly_dc_energy
          - sensor.monthly_inverter
      - entity_id: sensor.monthly_battenergycharged
        name: IN
        color: rgb(155,215,0)
  - entities:
      - entity_id: sensor.monthly_inverter
        name: INV
        color: var(--success-color)
        children:
          - sensor.monthly_energiatot
      - entity_id: sensor.monthly_dc_energy
        name: DC
        color: var(--warning-color)
        children:
          - sensor.monthly_energiatot
      - entity_id: sensor.monthly_energy
        color: PURPLE
        name: ENEL
        children:
          - sensor.monthly_energiatot
  - entities:
      - entity_id: sensor.monthly_energiatot
        name: TOT
card_mod:
  style:
    $: |
      h1.card-header {
          
          font-size: 18px;
          font-weight: normal;
          padding-bottom: 2px;
          padding-top: 2px;
          padding-left: 12px;
          padding-right: 12px;
        }
      ha-card{
         
         font-weight: 400;
         font-size: 13px;
         color: var(--text-color-off);
         padding: 0px 0px;
         margin: 0px 0px 0px 0px;
         }

```

![image](https://user-images.githubusercontent.com/67467751/211597302-0f79be78-c469-4463-98b0-928575c64ea8.png)

other exemple

```
type: custom:sankey-chart
title: Energia giornaliera
show_names: true
round: 1
unit_prefix: k
wide: true
min_box_distance: 2
min_box_height: 5
sections:
  - entities:
      - entity_id: sensor.daily_pvenergy
        name: FV1
        color: rgb(255,215,0)
        children:
          - sensor.daily_inverter

          - sensor.daily_dc_energy
          - sensor.daily_battenergycharged
#          - sensor.daily_battenergycharged
#          - sensor.daily_dc_energy
#          - sensor.daily_inverter
      - entity_id: sensor.daily_fvenergy
        name: FV2
        color: rgb(255,105,0)
        children:
          - sensor.daily_inverter

          - sensor.daily_dc_energy
          - sensor.daily_battenergycharged
      - entity_id: sensor.daily_battenelenergycharged
        name: ENEL IN
        color: purple
        children:
          - sensor.daily_battenergycharged
  - entities:
      - entity_id: sensor.daily_inverter
        type: passthrough
        children:
          - sensor.daily_inverter
      - entity_id: sensor.daily_dc_energy
        type: passthrough
        children:
          - sensor.daily_dc_energy
      - entity_id: sensor.daily_battenergydischarged
        name: OUT
        color: rgb(225,25,0)
        children:
          - sensor.daily_dc_energy
          - sensor.daily_inverter
      - entity_id: sensor.daily_battenergycharged
        name: IN
        color: rgb(155,215,0)
  - entities:
      - entity_id: sensor.daily_inverter
        name: INV
        color: var(--success-color)
        children:
          - sensor.energiatot
      - entity_id: sensor.daily_dc_energy
        name: DC
        color: var(--warning-color)
        children:
          - sensor.energiatot
      - entity_id: sensor.daily_energy
        color: PURPLE
        name: ENEL
        children:
          - sensor.energiatot
  - entities:
      - entity_id: sensor.energiatot
        name: TOT


```

![image](https://user-images.githubusercontent.com/67467751/211598358-eb92cae4-5a8d-4273-92a9-1b2de947f427.png)

if i change the orther of first child entities there aren t any graphical issue
```
type: custom:sankey-chart
title: Energia giornaliera
show_names: true
round: 1
unit_prefix: k
wide: true
min_box_distance: 2
min_box_height: 5
sections:
  - entities:
      - entity_id: sensor.daily_pvenergy
        name: FV1
        color: rgb(255,215,0)
        children:
          - sensor.daily_battenergycharged
          - sensor.daily_dc_energy
          - sensor.daily_inverter
      - entity_id: sensor.daily_fvenergy
        name: FV2
        color: rgb(255,105,0)
        children:
          - sensor.daily_battenergycharged
          - sensor.daily_dc_energy
          - sensor.daily_inverter
      - entity_id: sensor.daily_battenelenergycharged
        name: ENEL IN
        color: purple
        children:
          - sensor.daily_battenergycharged
  - entities:
      - entity_id: sensor.daily_inverter
        type: passthrough
        children:
          - sensor.daily_inverter
      - entity_id: sensor.daily_dc_energy
        type: passthrough
        children:
          - sensor.daily_dc_energy
      - entity_id: sensor.daily_battenergydischarged
        name: OUT
        color: rgb(225,25,0)
        children:
          - sensor.daily_dc_energy
          - sensor.daily_inverter
      - entity_id: sensor.daily_battenergycharged
        name: IN
        color: rgb(155,215,0)
  - entities:
      - entity_id: sensor.daily_inverter
        name: INV
        color: var(--success-color)
        children:
          - sensor.energiatot
      - entity_id: sensor.daily_dc_energy
        name: DC
        color: var(--warning-color)
        children:
          - sensor.energiatot
      - entity_id: sensor.daily_energy
        color: PURPLE
        name: ENEL
        children:
          - sensor.energiatot
  - entities:
      - entity_id: sensor.energiatot
        name: TOT
card_mod:
  style:
    $: |
      h1.card-header {
          
          font-size: 18px;
          font-weight: normal;
          padding-bottom: 2px;
          padding-top: 2px;
          padding-left: 12px;
          padding-right: 12px;
        }
      ha-card{
         
         font-weight: 400;
         font-size: 13px;
         color: var(--text-color-off);
         padding: 0px 0px;
         margin: 0px 0px 0px 0px;
         }

```

![image](https://user-images.githubusercontent.com/67467751/211598536-5f6d8605-d4df-4c1d-a177-32788d2571d1.png)

**Comment by MindFreeze** [comment]
I just fixed a bug with passthrough in v1.8.0 . Please try to reproduce after you update. It may be fixed

**Comment by lucasimons** [comment]
> I just fixed a bug with passthrough in v1.8.0 . Please try to reproduce after you update. It may be fixed

I tried but it seems to be the same.
![image](https://user-images.githubusercontent.com/67467751/211623928-ea1cdae5-a71c-4650-a3a9-ff8cb9c999b4.png)
